jschon/vocabulary/__init__.py

```diff
@@ -51,6 +51,22 @@ def _bootstrap(self, value: Mapping[str, JSONCompatible]) -> None:
             for vocabulary in self.default_vocabularies:
                 self.kwclasses.update(vocabulary.kwclasses)
 
+    def get_kwclass(self, key):
+        try:
+            return self.kwclasses[key]
+        except KeyError:
+            capitalized_name = (
+                key[0].upper() +
+                (key[1:] if len(key) > 1 else "")
+            )
+            unknown_class = type(
+                f'UnknownKeyword{capitalized_name}',
```

Comment:
There's a Python shortcut for this :wink: Check out `str.title()`

Reply:
I'm familiar with it, and sadly...
```Python
>>> "unknownKeyword".title()
'Unknownkeyword'
```
😭 
We need a `str.camel()` or something.

Reply:
Ah, gotcha. In that case the following will suffice:
```py
capitalized_name = key[0].upper() + key[1:]
```
A non-existent slice of a string just gives an empty string, there's no KeyError to avoid.